js/Worker.js

```diff
@@ -25,9 +26,15 @@ export default class Worker {
       },
     );
 
-    this.started = NativeModule.startWorker(
-      this.key, bundleRoot, bundleResource, parseInt(bundlerPort, 10)
-    );
+    this.started = new Promise(resolve => {
```

Comment:
is this our change?  if so, what's the reasoning?

Reply:
this is the upgrade.